src/distributions/range.rs

```diff
@@ -314,6 +314,19 @@ macro_rules! range_int_impl {
                 }
             }
         }
+
+        impl From<::core::ops::Range<$ty>> for RangeInt<$ty> {
```

Comment:
Can you also implement `From<::core::ops::Range<$ty>>` for `RangeFloat`?

Reply:
I'm not sure this is a good idea, since we don't have an implementation for closed ranges. On the other hand, this also affects `Range::new` and `Range::new_inclusive`, which are currently identical for floats.

Reply:
And here it's annoying that the `a..b` syntax is half-open — if it were closed it wouldn't be confusing using the syntax for sampling both float and int ranges. As it is though I don't know.